src/OmnichannelChatSDK.ts

```diff
@@ -1693,6 +1635,146 @@ class OmnichannelChatSDK {
         }
     }
 
+    public async getAgentAvailability(optionalParams: GetAgantAvailabilityOptionalParams = {}): Promise<any> { // eslint-disable-line @typescript-eslint/no-explicit-any
+        const requestId = uuidv4();
```

Comment:
`requestId` is already generated on instantiation of of the `OmnichannelChatSDK` class. You can simple reuse `OmnichannelChatSDK.requestId`

Reply:
updated